Program.cs

```diff
@@ -45,13 +46,15 @@ static Program()
             BasicConfigurator.Configure(new Hierarchy(),fileappender);
 
             AppDomain.CurrentDomain.AssemblyResolve += Program.OnAssemblyResolve;
-            if (!AppDomain.CurrentDomain.FriendlyName.EndsWith("vshost.exe"))
-            {
+            //if (!AppDomain.CurrentDomain.FriendlyName.EndsWith("vshost.exe"))
+            //{
+
                 AppDomain currentDomain = AppDomain.CurrentDomain;
                 currentDomain.UnhandledException +=
                     new UnhandledExceptionEventHandler(OnUnhandledException);
+                currentDomain.FirstChanceException += FirstChanceException;
                 Application.ThreadException += OnUnhandledThreadException;
-            }
+            //}
```

Comment:
maybe leave that in, it makes sure that visual studio / other debuggers handle the exceptions if its been launched from a debugger,  instead of any error handler from us, less interferences

Reply:
For some reason my debugger doesn't run with "vshost.exe". Maybe we should #if !DEBUG instead?